packages/prisma/extensions/disallow-undefined-in-delete-update-many.ts

```diff
@@ -0,0 +1,31 @@
+import { Prisma } from "@prisma/client";
+
+const checkUndefinedInValue = (where: any) => {
+  if (where) {
+    for (const key in where) {
+      // INFO: Since this is for $allModels, we don't have a way to get the correct
+      // where type
+      const whereInput = where[key as any] as any;
```

Comment:
I think better type here could be as `unknown` so that it forces us to do all possible checks on `whereInput` before using it.

Like `whereInput` should not be undefined and not null.

Reply:
Cool. Will switch.